pkg/odo/cli/config/set.go

```diff
@@ -74,13 +75,20 @@ func (o *SetOptions) Complete(name string, cmd *cobra.Command, args []string) (e
 	if o.now {
 		checkRouteAvailability = true
 	}
-	o.Context, err = genericclioptions.New(genericclioptions.CreateParameters{
+	var err2 error
+	o.Context, err2 = genericclioptions.New(genericclioptions.CreateParameters{
```

Comment:
I don't see why we are not using `err` here instead.

Reply:
Duplication warning by sonarcloud